apps/web/modules/survey/survey-editor/components/settings-view.tsx

```diff
@@ -44,7 +42,14 @@ export const SettingsView = ({
 
   return (
     <div className="mt-12 space-y-3 p-5">
-      <HowToSendCard localSurvey={localSurvey} setLocalSurvey={setLocalSurvey} environment={environment} />
+      <HowToSendCard
+        localSurvey={localSurvey}
+        setLocalSurvey={setLocalSurvey}
+        environment={{
```

Comment:
this can be changed to simple `environment={environment}` as they're the same type at both places